mod/carrierwave/lib/carrier_wave/file_card_uploader.rb

```diff
@@ -231,19 +235,6 @@ def storage
     end
   end
 
-  class SanitizedFile
-    def content_type
-      # the original content_type method doesn't seem to be very reliable
-      # It uses mime_magic_content_type  - which returns invalid/invalid for css files
-      # that start with a comment - as the second option.  (we switch the order and
-      # use it as the third option)
-      @content_type ||=
-        existing_content_type ||
-        mini_mime_content_type ||
-        mime_magic_content_type
-    end
-  end
-
```

Comment:
Maybe we have to reorder the new mime type recognitions methods here:
```
        identified_content_type ||
        declared_content_type ||
        guessed_safe_content_type ||
        Marcel::MimeType::BINARY
```

`identified` returns nil for wikirate's css, `guessed` returns `plain/css`. I don't know what `declared` returns

Reply:
Yeah, I realized as we drove away that this must be the problem area.